remove details on spent time from project overview

NobodysNightmare opened the pull request:
:warning: _Please note, that the requirements changed. See the [WP](https://community.openproject.org/work_packages/20219) for details. This PR should not be merged as long as it does not reflect those changes_
## OpenProject work package

https://community.openproject.org/work_packages/20219
## Description

This PR removes details on the spent time from the overview page of a project.
## Related PRs

This PR should be merged together with the following other PRs:
- https://github.com/finnlabs/openproject-my_project_page/pull/46
- https://github.com/finnlabs/openproject-reporting/pull/60

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://nobodysnightmare-feature-remove-reporting-details.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://nobodysnightmare-feature-remove-reporting-details.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

NobodysNightmare committed:
remove details on spent time from project overview

floriank commented:
Should we close this and it's adjacent PRs then? @NobodysNightmare

NobodysNightmare commented:
@floriank If I remember the new requirements correctly I can work on from this PR.

Basically they should now state: "still remove the stuff you removed, but add a replacement somewhere else".

NobodysNightmare committed:
add item to reach time entries to project menu

NobodysNightmare committed:
allow to hide menu items conditionally

NobodysNightmare committed:
deprecate method for deletion of menu items

floriank commented:
There is an "odd" addition to the modules menu, which i can't seem to find in the specification for the problem.

@NobodysNightmare could you check?

NobodysNightmare committed:
remove redundant localization

[ci skip]

NobodysNightmare commented:
The "odd" addition is expected and already present in the base branch.

However, that made me notice that I introduced duplicate translations, so I fixed those.

Note that we still need to wait for [Travis](https://travis-ci.org/opf/openproject/builds/68147455)
